src/store/model/mediamarkt-nl.ts

```diff
@@ -0,0 +1,51 @@
+import {Store} from './store';
+
+export const MediamarktNL: Store = {
+	labels: {
+		inStock: {
+			container: '.offers',
+			text: ['Online op voorraad']
```

Comment:
```suggestion
			container: 'div.price-button',
			text: ['bestel nu']
```

Isn't this what you'd want?

![image](https://user-images.githubusercontent.com/12074633/100934190-651fa200-34bc-11eb-9301-da7b8687448d.png)